### PR TITLE
Minor README update about proto

### PR DESCRIPTION
**Pull request by psfinaki**
It's Bootstrap actually.

image

### CODE REVIEW

File: DEVGUIDE.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ Running any of the above will build the latest changes and run tests against the
 
 ## Using your custom compiler to build this repository
 
-By removing all the subfolders called `Proto` under `artifacts` and running the `build` script again, the proto compiler will include your changes.
+By removing all the subfolders called `Bootstrap` under `artifacts` and running the `build` script again, the proto (= bootstrap) compiler will include your changes.
```
Comment (vzarytovskii) [inline DEVGUIDE.md:127]:
```suggestion
By removing all the subfolders called `Bootstrap` under `artifacts` and running the `build` script again, the proto compiler will include your changes.
```

Proto = compiler configuration in solution. Bootstrap = process.

Comment (psfinaki) [reply DEVGUIDE.md]: Yeah, correct. The one in /fsc is called Proto.
image

I put both names there then. As for proto vs bootstrap - would be useful to explain and uniform it. E.g. in the build script, `proto` is an alias for `bootstrap`.

But I just wanted to put the right folder name at least here :)

Comment (vzarytovskii) [reply DEVGUIDE.md]: `bootstrap` is a process, as in "we need to bootstrap Proto compiler", Proto is name of the configuration in solution.

File: DEVGUIDE.md [hunk 2]
```diff
@@ -124,7 +124,7 @@ Running any of the above will build the latest changes and run tests against the
 
 ## Using your custom compiler to build this repository
 
-By removing all the subfolders called `Proto` under `artifacts` and running the `build` script again, the proto compiler will include your changes.
+By removing all the subfolders called `Bootstrap` or `Proto` under `artifacts` and running the `build` script again, the proto compiler will include your changes.
```
Comment (abonie) [inline DEVGUIDE.md:127]: Is it ever `Proto`, or should it be just `Boostrap` here?

Comment (psfinaki) [reply DEVGUIDE.md:127]: It is Bootstrap in the artifacts root and then Proto in bin/fsc.

**Review by T-Gro — APPROVED** [review]

**Pull request merged**